test/fixtures/dialects/oracle/prompt.sql

```diff
@@ -0,0 +1,2 @@
+PROMPT this is an Oracle SQL newline delimited prompt statement
+SELECT job_id from employees;
```

Comment:
Since you specifically added a change for an empty PROMPT shall we add a test case for that?

This will also test that PROMPTS don't need to be semi-colon separated

```suggestion
PROMPT this is an Oracle SQL newline delimited prompt statement
PROMPT
PROMPT another prompt
SELECT job_id from employees;
```

Reply:
Whoops, sorry for the delay here. Thanks for picking up my slack @tunetheweb! I was skiing ⛷️ 😄